### PR TITLE
Remove HDT_Rarity

### PR DESCRIPTION
**Pull request by Jecoms**
The HDT Enums.Rarity.cs is removed in favor of using the
HearthDB.Enums.Rarity.

### CODE REVIEW

**Comment by jleclanche** [comment]
gtm

**Comment by Epix37** [comment]
@ko-vasilev Does this break the collection tracker plugin?

File: Hearthstone Deck Tracker/Utility/Themes/CardBarImageBuilder.cs [hunk 1]
```diff
@@ -209,7 +209,7 @@ protected virtual void AddCountBox()
 						countBox = OptionalCountBoxes[ThemeElement.EpicCountBox];
 						break;
 
-                    case Rarity.LEGENDARY:
+                    			case Rarity.LEGENDARY:
```
Comment (Epix37) [inline Hearthstone Deck Tracker/Utility/Themes/CardBarImageBuilder.cs]: Please squash this into the previous commit.

**Comment by Jecoms** [comment]
Well that didn't work as planned. Sorry I'm still learning the git commands. Will squash them down to one commit tonight. I can also mark the Rarity enum with the Obsolete annotation if you would like.

**Comment by Jecoms** [comment]
I'll resubmit with a single commit and the Obsolete Annotation.

**Pull request closed without merging**